src/constants/memory/wram.asm

```diff
@@ -3155,12 +3094,12 @@ wPaletteDataFlags::
 wPaletteToLoadForTileMap::
   ds 1 ; DDD2
 
-; TODO comment
-wPaletteUnknownC::
+; Palete color index to start copying palette data from if bit 7 is set in wPaletteDataFlags
+wPaletteParticalCopyColorIndexStart::
   ds 1 ; DDD3
 
-; TODO comment
-wPaletteUnknownD::
+; Amount of colors to copy if bit 7 is set in wPaletteDataFlags
+wPaletteParticalCopyColorCount::
```

Comment:
Same: is it `wPalettePartialCopyColorCount` instead?

```suggestion
wPalettePartialCopyColorCount::
```

Reply:
Arg, stupid typo, fixed it.